### PR TITLE
Update documentation files

### PR DESCRIPTION
**Pull request by curquiza**
- Update CONTRIBUTING.md
- Update the README -> this is not a WIP anymore 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,11 +57,11 @@ Add the following code to the `<dependencies>` section of your project:
 </dependency>
 ```
 
-### Gradle
+### Gradle <!-- omit in toc -->
 
 Add the following line to the `dependencies` section of your `build.gradle`:
 
-```
+```java
```
Comment (eskombro) [inline README.md:64]:
```suggestion
```groovy
```

**Review by eskombro — APPROVED** [review]
Thanks for this PR, great improvement in documentation.

Just added a suggestion as the syntax for the `build.gradle` file is determined by Groovy and not Java

🎉

**Review by eskombro — APPROVED** [review]
🌮

**Comment by curquiza** [comment]
bors merge

**Pull request merged**